V3-Upgrade-Guide.md

```diff
@@ -70,6 +70,19 @@ The options `versionInPackageJson` and `licenseInPackageJson`, that allowed sett
 When the `packageJson` option is enabled, a `package.json` file with version `1.0.0` and license `UNLICENSED` is created.
 If you want to set custom values, use the `include` option to add a custom `package.json`.
 
+The following deprecated options were removed with a clear replacement:
+- `forceOverwrite` becomes `overwrite`
+- `generatePackageJson` becomes `overwrite`
+- `writeReadme` becomes `readme`
+- `processesJsGeneration` becomes `transpilationProcesses`
+
+The deprecated `generateNpmrc` option of the generator is removed. 
+In most cases the `.npmrc` was present in the project root anyhow. 
```

Comment:
why? is this something the user needs to do? not clear to me.

Reply:
We used the .npmrc to set the repository url for all `@sap/` dependencies to the internal NPM this is not needed anymore. In general I think most of the options of the `npmrc` apply to the whole project so it is better to put it in the project root, instead of having configs in subsfolders which only apply if you execute the install in this folder. I will remive the sentence and just say -> use include now.